### DISCUSSION TITLE
Suggestion: can we have this syntax as a shorthand?

### DISCUSSION THREAD

**Discussion by anasouardini** [tailwindlabs/tailwindcss#10430] [Ideas]
I'm not sure about this tho, but the only downside I see is that `document.querySelector` does not accept this syntax.

instead of: `p-10 pb-6`.
it could be: `p(10 b-6)`

OR:

instead of: `before:bg-orange-500 before:text-white`.
it could be: `before(bg-orange-500 text-white)`.

again I'm not sure but I hope you take this into consideration. sorry if I missed something.

**Comment by RobinMalfait** [comment]
Hey! We get this question a lot and this won't work with that exact syntax because you can't have spaces in a class.

We wrote more about it here: https://twitter.com/adamwathan/status/1461519820411789314?s=46&t=6C9-VsVY7kdphAMiyPLKOA

**Comment by anasouardini** [comment]
oh thanks, and sorry for my noob question.

it doesn't work with all properties tho. can I have a reference to the docs for this feature?
